pkg/ocp/machinesets.go

```diff
@@ -95,7 +95,7 @@ func (msd *k8sMachineSetDeployer) clientForMsd(nameSpace string) dynamic.Resourc
 	return msd.dynamicClient.Resource(machinesetGVR).Namespace(nameSpace)
 }
 
-func (msd *k8sMachineSetDeployer) GetWorkerNodeImage(workerNodeList []string, machineSet *unstructured.Unstructured,
+func (msd *k8sMachineSetDeployer) GetWorkerNodeImage(workerNodeNames []string, machineSet *unstructured.Unstructured,
```

Comment:
```suggestion
func (msd *k8sMachineSetDeployer) GetWorkerNodeImage(machineSet *unstructured.Unstructured,
```

Better yet, drop this argument altogether because all callers even send is an empty slice.

This will simplify the function further.

Reply:
Indeed, thanks! I’ve added that as a second commit, so we have the variant with the node list in history if we ever need it.

Reply:
Sure makes sense from a commit history perspective :eyes: